playground/backend/internal/preparators/go_preparators.go

```diff
@@ -15,7 +15,30 @@
 
 package preparators
 
+import (
+	"errors"
+	"os/exec"
+	"path/filepath"
+)
+
 // GetGoPreparators returns reparation methods that should be applied to Go code
 func GetGoPreparators(filePath string) *[]Preparator {
-	return &[]Preparator{}
+	preparatorArgs := make([]interface{}, 1)
+	preparatorArgs[0] = filePath
+	formatCodePreparator := Preparator{Prepare: formatCode, Args: preparatorArgs}
+	return &[]Preparator{formatCodePreparator}
+}
+
+// formatCode formats go code
+func formatCode(args ...interface{}) error {
+	filePath := args[0].(string)
+	file := filepath.Base(filePath)
+	fileDir := filepath.Dir(filePath)
+	cmd := exec.Command("go", "fmt", file)
+	cmd.Dir = fileDir
+	stdout, err := cmd.CombinedOutput()
+	if err != nil {
+		return errors.New(string(stdout))
```

Comment:
I think we also need to return an error. May be something
```
return fmt.Errorf("%s: %s", like err.Error(), string(stdout))
```

Reply:
the stdout already contain this error